### PR TITLE
Add modeling.Tabular1D.inverse

### LINKED ISSUE #9070: Add inverse to Tabular1D model

### PR DESCRIPTION
**Pull request by jdavies-st**
Resolves #9070.

### CODE REVIEW

**Comment by bsipocz** [comment]
This is a new feature, isn't it? Then has to go to milestone 4.0.

**Comment by nden** [comment]
@bsipocz Ah, right. Thanks for clarifying!

**Comment by jdavies-st** [comment]
Some failing tests on CircleCI, but they look to be unrelated to this PR.

**Comment by pllim** [comment]
Hmm... `mpl` failures... @astrofrog or @bsipocz ?

**Comment by astrofrog** [comment]
@jdavies-st - could you rebase this? (this should get rid of the CircleCI failures)

**Comment by nden** [comment]
Looks good. Needs a rebase and a change log.

File: astropy/modeling/tests/test_models.py [hunk 1]
```diff
@@ -690,11 +689,46 @@ def test_tabular_with_bounding_box():
...
+    points = np.arange(5)
+    values = np.array([1.5, 3.4, 6.7, 7, 32])
+    t = models.Tabular1D(points, values)
+    result = t.inverse((3.4, 6.7))
+    assert_allclose(result, np.array((1., 2.)))
 
-    assert(t(1*u.pix, with_bounding_box=True) == 1.*u.AA)
+    # Check that it works for descending values in lookup_table
+    t2 = models.Tabular1D(points, values[::-1])
+    result2 = t.inverse((3.4, 6.7))
```
Comment (nden) [inline astropy/modeling/tests/test_models.py]: I think if you add a check here that `t.inverse.points = t.lookup_table[::1]` the code coverage test will pass

**Pull request merged**